internal-enrichment/virustotal/src/virustotal/client.py

```diff
@@ -54,6 +54,7 @@ def _query(self, url):
         adapter = HTTPAdapter(max_retries=retry_strategy)
         http = requests.Session()
         http.mount("https://", adapter)
+        self.headers["content-type"] = "application/json"
         response = None
         try:
             response = http.get(url, headers=self.headers)
```

Comment:
What about not modifying the `self.headers` dict at each request and add the `content-type` key directly in the `headers=` to avoid having to pop it each time in the `_post` method?
For example: 
```
response = http.get(url, headers=self.headers | {"content-type": "application/json"})
```

Reply:
Sounds like a better way to do it! Will fix and test